exes/mqtt-broadcaster/inc/mqtt_client.hpp

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+#include <openssl/ssl.h>
+#include <async_mqtt/buffer.hpp>
+#include <async_mqtt/endpoint.hpp>
+#include <async_mqtt/packet/qos.hpp>
+#include <async_mqtt/packet/v5_publish.hpp>
+#include <async_mqtt/predefined_underlying_layer.hpp>
+#include <async_mqtt/tls.hpp>
```

Comment:
change from optional to unique_ptr and than all of async_mqtt can be forward declared

Reply:
The architecture of this has changed, now a mqtt_client wrapper is used.